packages/react-is/src/ReactIs.js

```diff
@@ -68,8 +68,8 @@ export const StrictMode = REACT_STRICT_MODE_TYPE;
 
 export {isValidElementType};
 
-export function isAsyncMode(object: any) {
-  return typeOf(object) === REACT_ASYNC_MODE_TYPE;
+export function isConcurrentMode(object: any) {
```

Comment:
Both this and symbol export rename would be a breaking change technically. Unfortunately we forgot the unstable prefix here. Let’s keep both and add a deprecation warning for isAsyncMode.

Reply:
I've made the change. Did I do it correctly?